src/DataAccess/EntityManagerFactory.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\MembershipContext\DataAccess;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Doctrine\Common\EventManager;
+use Doctrine\Common\Persistence\Mapping\Driver\MappingDriverChain;
+use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
+use Doctrine\Common\Persistence\Mapping\Driver\MappingDriver;
+use Doctrine\DBAL\Connection;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Tools\Setup;
+use Gedmo\Timestampable\TimestampableListener;
+
+class EntityManagerFactory {
+
+	/**
+	 * Use this constant for MappingDriverChain::addDriver
+	 */
+	public const DOCTRINE_ADDITIONAL_ENTITIES = [
+		'WMDE\Fundraising\MembershipContext\DataAccess\DoctrineEntities' => __DIR__ . '/DoctrineEntities'
+	];
+
+	private const ENTITY_PATHS = [
+		__DIR__ . '/DoctrineEntities/'
+	];
+
+	private const ENTITY_NAMESPACE = 'WMDE\Fundraising\MembershipContext\DataAccess\DoctrineEntities';
+
+	private Connection $connection;
+	private string $proxyPath;
+	private AnnotationReader $annotationReader;
+
+	public function __construct( Connection $connection, string $proxyPath ) {
+		$this->connection = $connection;
+		$this->proxyPath = $proxyPath;
+		$this->annotationReader = new AnnotationReader();
+	}
+
+	public function newEntityManager( array $additionalEventSubscribers = [] ): EntityManager {
```

Comment:
This method (and `setupEventSubscribers`) should be moved to a subclass in the `Test` namespace, since the FF will create its own entity manager with the mappers and subscribers from the factory. The factory itself might then be renamed to `DoctrineSetupFactory` or smth